templates/around/map.html

```diff
@@ -3,10 +3,14 @@
 {% block title %}Around you{% endblock %}
 
 {% block extra_css %}
-  <link rel=stylesheet href=https://api.mapbox.com/mapbox-gl-js/v0.34.0/mapbox-gl.css >
-  <link rel=stylesheet href={{ url_for('static', filename='css/trips-map.css') }}>
   <link rel=stylesheet href={{ url_for('static', filename='css/around-map.css') }}>
-  <script src=https://api.mapbox.com/mapbox-gl-js/v0.34.0/mapbox-gl.js></script>
+  <!-- leaflet -->
+  <link rel="stylesheet" href="https://unpkg.com/leaflet@1.2.0/dist/leaflet.css" integrity="sha512-M2wvCLH6DSRazYeZRIm1JnYyh22purTM+FDB5CsyxtQJYeKq83arPe5wgbNmcFXGqiSH2XR8dT/fJISVA1r/zQ==" crossorigin=""/>
```

Comment:
you very obviously copy and pasted. Code I've never seen before 😮 
You may remove quotes where not required ;)